JavaScript/question/app.js

```diff
@@ -0,0 +1,185 @@
+
+/* Q1  */
+let nickname ='ごっしー';
+let age=28;
+
+console.log('私のニックネームは' + nickname + 'です。' + '年齢は' + age + '歳です。' );
+
+/* Q2 */
+let languages = ['JavaScript' ,'PHP' ,'Ruby' ,'Python' , 'GO'];
```

Comment:
ほとんどのプログラミング言語では、配列を定義するとき`,`の位置は以下のようになりますよ
```suggestion
let languages = ['JavaScript', 'PHP', 'Ruby', 'Python', 'GO'];
```

ただ配列は横に長くなりやすいので、ほとんどの場合は縦にしますね
```suggestion
let languages = [
  'JavaScript',
  'PHP',
  'Ruby',
  'Python',
  'GO'
];
```

Reply:
そうなんですね